frontend/src/Frontend/UI/Transfer.hs

```diff
@@ -461,9 +463,57 @@ lookupKeySets logL networkName nodes chain accounts = do
           putLog logL LevelInfo $ "lookupKeysets failed:" <> tshow e
           pure Nothing
 
-    liftIO $ trigger r
+  -- TODO: CLEAN THIS UP!
+    rr <- case r of
+      Just res -> fmap Just $ iforM res $ \(AccountName name) dets -> do
+        let ref' = dets ^? _AccountStatus_Exists . accountDetails_guard . _AccountGuard_KeySetRef
+        let bal' = dets ^? _AccountStatus_Exists . accountDetails_balance . (to unAccountBalance)
+            updateBal old new = if old == new then old else new
+        case (,) <$> bal' <*> ref' of
+          Just (bal, ref) -> do
+            let describeCode keysetref account =
+                  [NeatInterpolation.text|{"balance" : (at 'balance (coin.details "$account")), "guard" : (describe-keyset "$keysetref")}|]
```

Comment:
This is not a very big string literal, so let's just use `printf` and avoid the dependency on `neat-interpolation`.

Reply:
OK, using `printf` now.